source/usage.rst

```diff
@@ -452,3 +452,45 @@ to the container if it does not stop. But it slows the tests needlessly down, as
 the container runtime waits for 10 seconds before sending
 ``SIGKILL``. Therefore, ``pytest_container`` sets the stop signal to
 ``SIGTERM``, if used :file:`/bin/bash` as the entrypoint.
+
+
+Using pytest marks for filtering tests
+--------------------------------------
+
+Pytest supports adding custom marks to tests or test parameters for filtering
+tests for test runs. For a general introduction to marking tests, refer to the
+upstream documentation:
+<https://docs.pytest.org/en/latest/example/markers.html#mark-examples>_
+
+It can be beneficial to mark each container for a test suite with a unique
+marker, so that you can run the tests only for a single container image. This
+would involve wrapping each :py:class:`~pytest_container.container.Container`
+and :py:class:`~pytest_container.container.DerivedContainer` in a
+``pytest.param()``. This approach two disadvantages:
+
+1. you are now handling ``pytest.ParameterSet`` instance and have to unpack them
+   to access the underlying :py:class:`~pytest_container.container.Container`
+   and :py:class:`~pytest_container.container.DerivedContainer`
+
+2. :py:class:`~pytest_container.container.DerivedContainer` do not "inherit" the
+   marks from their base containers. This has proven to be a very annoying
+   limitation in practice.
```

Comment:
```suggestion
2. :py:class:`~pytest_container.container.DerivedContainer` does not "inherit" the
   marks from their base containers. This has proven to be a very annoying
   limitation in practice.
```

> "inherit"

Why inherit is in double quotes?

> This has proven to be a very annoying limitation in practice.

Either explain why it's a limitation (without using subjective "Annoying"), or drop it altogether.